packages/strapi-admin/admin/src/translations/cs.json

```diff
@@ -0,0 +1,215 @@
+{
+  "Analytics": "Analýzy",
+  "Content Manager": "Správce obsahu",
+  "Content Type Builder": "Tvůrce typů obsahu",
+  "Email": "E-mailová adresa",
+  "Files Upload": "Nahrát soubory",
+  "HomePage.notification.newsLetter.success": "Přihlášení k odběru novinek proběhlo úspěšně",
+  "New entry": "Nový záznam",
+  "Password": "Heslo",
+  "Provider": "Poskytovatel",
+  "ResetPasswordToken": "Token pro obnovu hesla",
+  "Role": "Role",
+  "Roles & Permissions": "Role a oprávnění",
+  "Settings Manager": "Správce nastavení",
+  "Username": "Uživatelské jméno",
+  "Users": "Uživatelé",
+  "Users & Permissions": "Uživatelé a oprávnění",
+  "app.components.BlockLink.code": "Příklady kódu",
+  "app.components.BlockLink.code.content": "Naučte se testovat skutečné projekty vyvinuté komunitou.",
```

Comment:
"Naučte se na příkladech vytvořených komunitou" looks better to me

Reply:
changed